ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: 3scale-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 3scale-amp-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      AWS_REGION: us-east-2
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      DEPL_BUCKET_NAME: 3scale-apimanager-s3-bucket
+      DEPL_PROJECT_NAME: threescale
+      OPERATORS: |
+        [
+            {"name": "3scale-operator", "source": "redhat-operators", "channel": "threescale-2.13", "install_namespace": "threescale", "target_namespaces": "threescale"}
+        ]
+      SPOT_INSTANCES: "true"
+    post:
+    - ref: gather-aws-console
```

Comment:
whats this ref for? Data/result collection? if so how to access those

Reply:
This ref is added in [ipi-aws workflow](https://github.com/openshift/release/blob/be8c4512b32e6a5125d972046aaf755ce0507b7a/ci-operator/step-registry/ipi/aws/ipi-aws-workflow.yaml#L7) as a [post chain](https://github.com/openshift/release/blob/be8c4512b32e6a5125d972046aaf755ce0507b7a/ci-operator/step-registry/ipi/aws/post/ipi-aws-post-chain.yaml#L4) and [here](https://github.com/openshift/release/blob/be8c4512b32e6a5125d972046aaf755ce0507b7a/ci-operator/step-registry/gather/aws-console/gather-aws-console-ref.yaml#L19) is the documentation of this ref. 

> The pre-deprovision artifacts step collects console logs from AWS instances.  It gathers console logs for all nodes and machines with a provider ID, as well as any instance IDs listed in ${SHARED_DIR}/aws-instance-ids.txt.  aws-instance-ids.txt entries should have a single provider ID per line.  Duplicate entries are ok; the step deduplicates before gathering.

You can access these logs from artifacts of prow job, like for this job here: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36871/rehearse-36871-periodic-ci-3scale-qe-3scale-deploy-main-3scale-amp-ocp4.13-lp-interop-3scale-amp-interop-aws/1638098496814845952/artifacts/3scale-amp-interop-aws/gather-aws-console/

Reply:
alright Thanks!